editor.html

```diff
@@ -109,6 +109,41 @@ <h3>Controls</h3>
             Show ghost image
           </label>
         </section>
+        <section class="sizes text">
+          <h3>Sizes</h3>
+          <label class="mask__option">
+            <input type="checkbox" name="x1" autocomplete="off" checked/>
```

Comment:
The checkboxes are all related so they should have the same name. You can keep individual values to differentiate them.
```suggestion
            <input type="checkbox" name="sizes" value="x1" autocomplete="off" checked/>
```

Reply:
Updated it with your suggestion